src/QueryDescription/index.js

```diff
@@ -247,22 +250,32 @@ export function where(left: ColumnName, valueOrComparison: Value | Comparison):
   return { type: 'where', left: checkName(left), comparison: _valueOrComparison(valueOrComparison) }
 }
 
-const acceptableClauses = ['where', 'and', 'or', 'on']
-const isAcceptableClause = (clause: Where) => acceptableClauses.includes(clause.type)
+export function unsafeSqlExpr(sql: string): SqlExpr {
+  invariant(typeof sql === 'string', 'Value passed to Q.unsafeSqlExpr is not a string')
+  return { type: 'sql', expr: sql }
+}
 
-export function and(...clauses: Where[]): And {
+export function unsafeLokiExpr(expr: any): LokiExpr {
+  invariant(expr && typeof expr === 'object', 'Value passed to Q.unsafeLokiExpr is not an object')
```

Comment:
This also allows to pass an Array. Is it ok?

Reply:
fixed